.github/workflows/ci.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
           node-version: '18.x'
 
       - name: Install dependencies
-        run: yarn --ignore-optional
+        run: npm i --omit optional
```

Comment:
Should we be using things like `npm ci` instead? It enforces not overwriting the lock file so changes need to be updated in there

Reply:
Updated!